vmm/src/config.rs

```diff
@@ -357,6 +357,10 @@ pub enum ValidationError {
     InvalidIvshmemPath,
     #[error("Payload configuration is not bootable")]
     PayloadError(#[from] PayloadConfigError),
+    #[error("mask provided without an ip")]
```

Comment:
```suggestion
    #[error("mask provided without an IP")]
```

Reply:
In our CHV code base, yes! Strictly speaking, Rust's guidelines say to use non-capital letters. However, this is another story and we should stay consistent :)